prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html

```diff
@@ -5,11 +5,11 @@
   <ng-container *ngIf="certificate; else noCertificate">
 
     <app-page-subheader>
-      <ng-container appPageSubheaderTitle>PharmaNet ID</ng-container>
+      <ng-container appPageSubheaderTitle>Global PharmaNet ID</ng-container>
     </app-page-subheader>
 
     <p class="mb-3">
-      The user's PharmaNet ID number (GPID) is:
+      The user's Global PharmaNet ID number (GPID) is:
```

Comment:
Should "number" be removed?

Reply:
Reviewed the UI with Nelson but did not catch that. "Number" is removed.